pkg/server/commands/list_objects_test.go

```diff
@@ -33,15 +36,19 @@ func TestNewListObjectsQuery(t *testing.T) {
 func TestListObjectsDispatchCount(t *testing.T) {
 	ds := memory.New()
 	ctx := storage.ContextWithRelationshipTupleReader(context.Background(), ds)
-
+	ctrl := gomock.NewController(t)
+	mockThrottler := mocks.NewMockThrottler(ctrl)
+	t.Cleanup(ds.Close)
```

Comment:
`t.Cleanups` are called in LIFO order, so should be written just after the resource was created:

```go
ds := memory.New()
t.Cleanup(ds.Close)

s := NewServer(...)
t.Cleanup(s.CLose)
```